src/craterstats/cli.py

```diff
@@ -6,6 +6,10 @@
 import argparse
 import numpy as np
 import re
+import pathlib
+
+here = str(pathlib.Path(__file__).parent.resolve())
```

Comment:
This is not a robust pattern for packaging. Sometimes the distribution will be compressed into an archive format like .egg or .whl for Python, and there is no file on disk to be resolved. The preferred method is to use the pkg_resources module from setuptools (Python 3.6) or importlib.resources (Python 3.7+).

Reply:
I elected to go for the importlib resources option by adding the backport package as a dependency for python 3.6 to avoid an runtime dependency on setuptools, but this required updating a few other files like the demo script, that said the demo works nicely